e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -91,45 +92,79 @@ describe("scenarios > dashboard > tabs", () => {
     });
   });
 
-  it("should allow moving dashcards between tabs", () => {
-    visitDashboardAndCreateTab({
-      dashboardId: ORDERS_DASHBOARD_ID,
-      save: false,
-    });
+  it(
```

Comment:
The "Move to tab" dropdown probably should not be shown when editing a link card:


https://github.com/metabase/metabase/assets/6830683/c123fbe3-2c7b-42fc-ad76-bd6090749d27

Reply:
@npretto :arrow_up:

Reply:
@kamilmielnik another solution would be to make it appear on top of the other menu (will ask product today)
<img width="448" alt="image" src="https://github.com/metabase/metabase/assets/1914270/015326e6-2ecc-4565-808b-61fbe60bc656">
but it involves touching zIndexs and it would make sense to fix it in M3 with the other zIndex issue.

Do you think we could merge it as it is and fix this too in M3?

Reply:
> Do you think we could merge it as it is and fix this too in M3?

Probably M3 but I'd double-check with product.

Reply:
Also moved to M3 ✅